assets/i18n/en.toml

```diff
@@ -4,26 +4,26 @@ docs = "Documentation"
 blog = "Blog"
 github = "GitHub"
 login = "Vehicle logins"
-about = "About evcc"
+about = "About"
 
 [header.theme]
 auto = "Design: system"
 light = "Design: light"
 dark = "Design: dark"
 
 [footer.version]
-availableShort = "update"
-availableLong = "update available"
-modalTitle = "Update available"
-modalUpdateStarted = "Evcc will restart after the update.."
-modalInstalledVersion = "Currently installed version"
-modalNoReleaseNotes = "No release notes available. More information about the new version can be found here:"
+availableShort = "install"
+availableLong = "new version available"
+modalTitle = "New version available"
+modalUpdateStarted = "Restarting after installing the new version…"
+modalInstalledVersion = "Installed version: "
+modalNoReleaseNotes = "No release notes available. More info about the new version:"
 modalCancel = "Cancel"
-modalUpdate = "Update"
-modalUpdateNow = "Update now"
+modalUpdate = "Install"
+modalUpdateNow = "Install now"
 modalDownload = "Download"
-modalUpdateStatusStart = "Update started: "
-modalUpdateStatusFailed = "Update failed: "
+modalUpdateStatusStart = "Installation started: "
+modalUpdateStatusFailed = "Installation failed: "
```

Comment:
```suggestion
modalUpdateStatusFailed = "Installation failed:"
```

Reply:
just saw that this key is not in use any more. I'll remove it.